internal/cmds/cmds.go

```diff
@@ -141,9 +151,11 @@ func uninstall(ctx *log.Context, h types.HandlerEnvironment, report *types.RunCo
 		ctx = ctx.With("path", constants.DataDir)
 		ctx.Log("event", "removing data dir", "path", constants.DataDir)
 		if err := os.RemoveAll(constants.DataDir); err != nil {
+			extensionEvents.LogErrorEvent("uninstall", "failed to delete data directory")
```

Comment:
Please add the directory and error message

Reply:
added